astropy/modeling/core.py

```diff
@@ -1110,17 +1110,16 @@ def render(self, out=None, coords=None):
             # Check dimensions match out and model
             assert len(coords) == ndim
```

Comment:
Happy to change, but you've suggested to keep this one before, https://github.com/astropy/astropy/pull/5580#pullrequestreview-12306818